### DISCUSSION TITLE
Export .json

### DISCUSSION THREAD

**Discussion by ka11zer** [ente-io/ente#2298] [Enhancements]
Please add support to export as .json instead of .txt

**Comment by ctrlcmdshft** [comment]
Please add this

**Comment by phucngo-work** [comment]
Really need this feature!

**Comment by thisguybikash** [comment]
I am trying to use Ente-CLI with Raycast, and Raycast requires a JSON export file for some reason. And I can't get the JSON file using Ente-CLI. Please add this feature, pretty please.

**Reply by vishnukvmd** [reply]
Could you follow the setup instructions here? https://www.raycast.com/chkpwd/ente-auth

**Reply by ctrlcmdshft** [reply]
Why can’t you just add the export option

**Comment by MikeSesketcharba** [comment]
Also want this! A little backwards this isn't even an option
